sabnzbd/interface.py

```diff
@@ -1555,9 +1555,9 @@ def index(self, **kwargs):
         )
         for svr in server_names:
             new.append(servers[svr].get_dict(safe=True))
-            t, m, w, d, timeline = sabnzbd.BPSMeter.amounts(svr)
+            t, m, w, d, timeline, bps = sabnzbd.BPSMeter.amounts(svr)
```

Comment:
We can ignore the output here `t, m, w, d, timeline, _ = sabnzbd.BPSMeter.amounts(svr)`